<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleNumberSet</class>
 <widget class="QWidget" name="VehicleNumberSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}</string>
  </property>
  <widget class="QPushButton" name="btnA1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>A1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>原列车号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>新列车号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginNumber">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewNumber">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: blue;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnB1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>B1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnC1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnD1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnA2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>A2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnB2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>B2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnC2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnD2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrong">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
color: red;</string>
   </property>
   <property name="text">
    <string>警告：车号必须是三位数！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
